### PR TITLE
added the alternative step as a note for editing topics

### PR DESCRIPTION
**Pull request by fmcdonal**

### CODE REVIEW

**Comment by fmcdonal** [comment]
HTML generated: http://file.rdu.redhat.com/~fmcdonal/JK-7879/README.html
@mmurphy and @jbyrne-redhat  please review for correctness and consistency similar to consumer groups.

File: docs/kafka/topic-configuration-kafka/README.adoc [hunk 1]
```diff
@@ -90,13 +90,14 @@ For a list of topic properties that you can update using the CLI, see the `rhoas
...
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the topics that you want to configure.
-. In the *Topics* page, click the name of the topic that you want to review or edit.
-+
-Alternatively, select the options icon (three vertical dots) for the relevant topic and click *Edit*.
-. Click *Edit properties*, review the current topic properties, and adjust any editable topic properties as needed.
+. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances*.
+. Click the Kafka instance *Name* that contains the topic you want to configure.
+. Select the *Topics* page.
```
Comment (jbyrne-redhat) [inline docs/kafka/topic-configuration-kafka/README.adoc:95]: Suggest: "Click the *Topics* tab.

Reasoning: The _tab_ is the UI element that you actually click. The page is the result.

File: docs/kafka/topic-configuration-kafka/README.adoc [hunk 2]
```diff
@@ -90,13 +90,14 @@ For a list of topic properties that you can update using the CLI, see the `rhoas
...
+. Select the options icon (three vertical dots) for the relevant topic *Name* and click *Edit* to review the current topic properties, and adjust any editable topic properties as needed.
 . Click *Save* to finish.
 
+NOTE: You can also edit a specific topic from your Kafka instance. Select the *Topics* page and click the name of the Kafka instance that contains the topic that you want to configure. Select the *Properties* page, and click *Edit properties*. Review the current topic properties, adjust any editable topic properties and click *Save*.
```
Comment (jbyrne-redhat) [inline docs/kafka/topic-configuration-kafka/README.adoc:99]: This isn't right. You click a topic on the *Topics* page, not a Kafka instance.

In any case, I think something shorter is sufficient. Something like:

NOTE: You can also edit topic properties by selecting the *Properties* tab within the topic.

File: docs/kafka/topic-configuration-kafka/README.adoc [hunk 3]
```diff
@@ -90,13 +90,14 @@ For a list of topic properties that you can update using the CLI, see the `rhoas
...
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the topics that you want to configure.
-. In the *Topics* page, click the name of the topic that you want to review or edit.
-+
-Alternatively, select the options icon (three vertical dots) for the relevant topic and click *Edit*.
-. Click *Edit properties*, review the current topic properties, and adjust any editable topic properties as needed.
+. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances*.
+. Click the Kafka instance *Name* that contains the topic you want to configure.
+. Select the *Topics* page.
+. Select the options icon (three vertical dots) for the relevant topic *Name* and click *Edit* to review the current topic properties, and adjust any editable topic properties as needed.
```
Comment (jbyrne-redhat) [inline docs/kafka/topic-configuration-kafka/README.adoc:96]: Suggest: "Select the options icon (three vertical dots) for the relevant topic **next to** the relevant topic....

File: docs/kafka/topic-configuration-kafka/README.adoc (hunk 1 above)
Comment (fmcdonal) [reply docs/kafka/topic-configuration-kafka/README.adoc:95]: I like tab too but other docs refer to these tabs as pages in the step previous to getting to the pages.

File: docs/kafka/topic-configuration-kafka/README.adoc [hunk 4]
```diff
@@ -90,13 +90,14 @@ For a list of topic properties that you can update using the CLI, see the `rhoas
 * You have created a Kafka instance with at least one Kafka topic in {product-kafka}.
 
 .Procedure
-. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that contains the topics that you want to configure.
-. In the *Topics* page, click the name of the topic that you want to review or edit.
-+
-Alternatively, select the options icon (three vertical dots) for the relevant topic and click *Edit*.
-. Click *Edit properties*, review the current topic properties, and adjust any editable topic properties as needed.
+. In the {product-kafka} web console, go to *Streams for Apache Kafka* > *Kafka Instances*.
+. Click the Kafka instance *Name* that contains the topic you want to configure.
```
Comment (jbyrne-redhat) [inline docs/kafka/topic-configuration-kafka/README.adoc:94]: I think we can drop **Name** here. So, just "Click the Kafka instance that contains..."

File: docs/kafka/topic-configuration-kafka/README.adoc [hunk 5]
```diff
@@ -90,13 +90,14 @@ For a list of topic properties that you can update using the CLI, see the `rhoas
...
+. Select the *Topics* tab.
+. Select the options icon (three vertical dots) for the relevant topic next to the relevant topic *Name* and click *Edit* to review the current topic properties, and adjust any editable topic properties as needed.
```
Comment (jbyrne-redhat) [inline docs/kafka/topic-configuration-kafka/README.adoc:96]: Can get rid of "next to the relevant topic **Name**"

**Pull request merged**